.circleci/config.yml

```diff
@@ -171,6 +202,25 @@ jobs:
       - store_artifacts:
           path: logs_cpp_parametric_dev.tar.gz
 
+  fuzz-testing:
+    docker:
+    - image: "datadog/docker-library:dd-trace-cpp-ci"
+    resource_class: xlarge
+    environment:
+      MAKE_JOB_COUNT: 8
+    steps:
+      - checkout
+      - run: bin/with-toolchain llvm cmake . -B .build -DCMAKE_BUILD_TYPE=Debug -DDD_TRACE_BUILD_FUZZERS=1 -DDD_TRACE_ENABLE_SANITIZE=1
+      - run: cmake --build .build -j ${MAKE_JOB_COUNT} --target dd_trace_cpp-fuzzers
+      - run_timeout: 
+          description: Run W3C propagation fuzzer
+          command: ./.build/fuzz/w3c-propagation/w3c-propagation-fuzz
+          duration: 5m
+      - run: 
```

Comment:
Any reason not to use `run_timeout` here? Or maybe merge mistake?

Reply:
That's a mistake. Thank you!

Reply:
```suggestion
      - run_timeout: 
```